views/layout-form-background.twig

```diff
@@ -38,63 +38,75 @@
 
 {% block formHtml %}
     <div class="row">
+        <ul class="nav nav-tabs" role="tablist">
+            <li class="active"><a href="#general" role="tab" data-toggle="tab"><span>{% trans "General" %}</span></a></li>
+            <li><a href="#actionTab" role="tab" data-toggle="tab"><span>{% trans "Action" %}</span></a></li>
```

Comment:
We can probably add this one through the back end as you have done here (and the Region one) as that does not have any knock on implications for Custom stuff - I think?